dushimeemma/fs-simba-clockwork
# CWM-3 Events endpoints

Description 

This PR introduces APIs Endpoints

Type of change 

[ ] Bug fix (Non breaking change that fixes an existing issue)

[x] New Feature (Non breaking change which adds funcitonality)

[ ] Breaking change (New feature that removes or causes an existing feature not to work as expected)

[ ] This change requires a documentation update

How Has This Been Tested? 

[x] Unit tests

[x] Component Tests

[x] Integration Tests

How Should This Be Manually tested:

- Clone this repo git clone https://github.com/dushimeemma/fs-simba-cal/.

- Checkout the branch git checkout branch_name

- Run yarn install

- Run yarn dev

Checklist: 

[x] My code follows all the styles and conventions outlined for this project

[x] I have linted my code before submission

[x] I have made my corresponding changes to the documentation

[x] My changes generates no new warning

[x] I have added tests that prove my feature is working or my fix is effective

[x] New and existing tests pass locally with my changes

JIRA 
[CWM-3](https://www.notion.so/APIs-Endpoints-91d9bd6fdc8b4f7aa009a66588c11269)
